packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -52,6 +52,7 @@ export interface ISiteProps {
     activeFramework?: Framework;
     collapsed?: boolean;
     componentBackgroundTransparent?: boolean;
+    showComponentStatus?: boolean;
```

Comment:
Technically this will disappear from anyone's site that currently has it implemented. If they don't want that, this is great...if they are using it, they will need to figure out why it's gone. 

@nicholasrice, @janechu should we default this to true and have it operate as "opt-out" to maintain current behavior?

Reply:
good call - yea I would default this to true

Reply:
Done